package.json

```diff
@@ -39,8 +39,10 @@
     "@apollo/icons": "^0.0.8",
     "@apollo/tailwind-preset": "^0.1.3",
     "@headlessui/react": "^1.7.17",
+    "@headlessui/tailwindcss": "^0.2.0",
     "@radix-ui/react-tabs": "^1.0.2",
     "clsx": "^2.1.0",
+    "framer-motion": "^11.0.3",
```

Comment:
Framer motion seems to be the only one that I can get proper [exit animations](https://www.framer.com/motion/animate-presence/). I could not get this using the built-in [`Transition` component](https://headlessui.com/react/transition) from [headless UI](https://headlessui.com/).